packages/analytics/analytics-chart/src/components/chart-plugins/ChartTooltip.vue

```diff
@@ -208,7 +208,7 @@ function handleMouseUp() {
 }
 .tooltip-container {
   background-color: $kui-color-background;
-  border-radius: 3px;
+  border-radius: var(--kui-border-radius-20, $kui-border-radius-20);
   box-shadow: 0 5px 15px rgba(0, 0, 0, 0.12), 0 5px 10px rgba(0, 0, 0, 0.24);
```

Comment:
I assume this is fine to leave as-is, since it's just the tooltip plugin's drop shadow.  Plus we can't do transparency with our color palette (right? 🤔 )